release_base_script.sh

```diff
@@ -26,6 +26,4 @@ if [[ "${GIT_BRANCH}" = "master" ]]; then
     VERSION=${VERSION:-"$(git describe --abbrev=0 --tags)"}
 elif [[ "${GIT_BRANCH}" = "develop" ]]; then
     VERSION="develop"
```

Comment:
It looks like the original code:
```
else
    exit 0
```
has been removed. If it was on purpose, it's fine. I just wanted to make sure that it was intended. Thanks.

Reply:
So, you seem to have deleted the else condition when you reset the changes. Please see the diff in GitHub when you get a chance. I was talking about that in my message above.

Reply:
The changes look good. Resolving this conversation. Thanks.